static/styles/subscriptions.css

```diff
@@ -141,22 +141,6 @@
     margin-right: 10px;
 }
 
-.preview-stream {
-    display: none;
-    float: right;
-    padding: 3px 10px;
-    border: 1px solid hsl(0, 0%, 80%);
-    margin: 9px 10px 0 0;
-    color: hsl(0, 0%, 20%);
-}
-
-.preview-stream:hover {
-    color: hsl(0, 0%, 20%);
-    text-decoration: none;
-    background-color: hsl(0, 0%, 92%);
-    border: 1px solid hsl(0, 0%, 68%);
-}
-
```

Comment:
Removed `.preview-stream` class as there's no such class present in the whole code.

Attaching a Screenshot for ease to check ->

![Screenshot from 2022-02-27 23-08-09](https://user-images.githubusercontent.com/76561593/155893734-81d8bb72-0ec4-4e7d-bab4-c44a519c1e67.png)